artemis-server/src/test/java/org/apache/activemq/artemis/core/paging/cursor/impl/PageReaderTest.java

```diff
@@ -58,7 +58,7 @@ public void testPageReadMessage() throws Exception {
             PagePosition pagePosition = new PagePositionImpl(10, i);
             pagedMessage = pageReader.getMessage(pagePosition);
          } else {
-            int nextFileOffset = pagedMessage == null ? -1 : offsets[i - 1] + pagedMessage.getEncodeSize() + Page.SIZE_RECORD;
+            int nextFileOffset = pagedMessage == null ? -1 : offsets[i - 1] + pagedMessage.getStoredSize() + Page.SIZE_RECORD;
```

Comment:
I don't know why, but my IDEA isn't happy with this test, see
![image](https://user-images.githubusercontent.com/13125299/76842129-1486ec80-683a-11ea-8c10-3d48418c9bdf.png)
Although I can run it and it compiles correctly...:(

Reply:
PageReaderTest is from a different testsuite. IDEA may not like it.

Reply:
It's not that simple to move this test. it uses a lot of package private methods.. moving would need to make them public or a bigger refactoring.

I had a version where I did this at some point. I would go for the simpler version now in case we ever need to cherry-pick to a point release.

Reply:
> PageReaderTest is from a different testsuite. IDEA may not like it.

What's strange is that I rememeber while doing it the same (basically same code) IDEA wasn't complaining that much, but is really a minor eh, can be ignored